### PR TITLE
Add Megan's bio

### PR DESCRIPTION
**Pull request by mmore500**

### CODE REVIEW

File: _people/megan_friedman.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Megan Friedman
+role: Lab Manager
+current: True
+image: "assets/images/people/megan_friedman.jpeg"
```
Comment (zamanlh) [inline _people/megan_friedman.md:5]: I hate when I do that!

**Review by zamanlh — APPROVED** [review]
Looks good!

**Comment by mmore500** [comment]
Aaaa! I got the last name wrong, right in the middle of fixing it. Second pull request in a second!

**Comment by zamanlh** [comment]
Ah! I can change that quickly!

Luis

**Pull request merged**